### ISSUE TITLE
forceClearScrollbackBuffer conflicts with focusConsoleOnExecute

### ISSUE THREAD

**Issue by BBengtsson** [PowerShell/vscode-powershell#2833]
Adding the following setting `"powershell.integratedConsole.forceClearScrollbackBuffer":true` causes the setting `"powershell.integratedConsole.focusConsoleOnExecute": false` to be ignored.

This means that the focus shifts to the terminal instead of staying inside the code window

### System Details

System Details Output
```

```

### Issue Description

I am experiencing a problem with keeping the focus on the code window instead of the terminal windows

#### Expected Behaviour

When i run/execute powershell code i expect the focus to stay in the code windows when this setting is added :
`"powershell.integratedConsole.focusConsoleOnExecute": false`

#### Actual Behaviour
The setting `"powershell.integratedConsole.focusConsoleOnExecute": false` is being ignored when i added `"powershell.integratedConsole.forceClearScrollbackBuffer": true,`, so everytime i execute code my cursor/marker is focused to the terminal windows.

### Attached Logs

Follow the instructions in the [troubleshooting docs](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#logs)
about capturing and sending logs.

**Comment by SydneyhSmith** [comment]
I am thinking @vexx32 was having this issue, but I cant seem to find the bug

**Comment by vexx32** [comment]
I don't think this was the same thing I was seeing, though my issue did also concern how the console focusing logic worked (it was revealing + focusing the wrong console when I had it hiding the integrated console by default and also running debug sessions in new consoles every time, forget the exact setting names but yeah).

**Comment by BBengtsson** [comment]
Can someone please have a look at this? It would really benefit my work to be able to clear the history the same way as powershell ISE and be able to keep focus on the code window at the same time. Been waiting for a fix for this since last summer.... =(
